src/platform/visualc/config.h

```diff
@@ -68,14 +74,8 @@
 /* Enable the FPU module, still only for beta testing */
 #define C_FPU 1
 
-/* Define to 1 to use a x86 assembly fpu core */
-#ifdef _M_X64
-//No support for inline asm with visual studio in x64 bit mode.
-//This means that non-dynamic cores can't use the better fpu emulation.
-#define C_FPU_X86 0
-#else // _M_IX86
+/* Define to 1 to use x86 assembly fpu core. Requires Clang toolchain for x64 */
 #define C_FPU_X86 1
-#endif
```

Comment:
Perhaps a `defined(__clang__)` check should go here instead of that hard error above.

Reply:
I'm in favour of the hard error. If we ship Clang builds, and the non-Clang build is gimped, re-enabling the gimped build should be only doable by manual hackery at one's own peril.

But we should _not_ actively encourage it. Quite the opposite: no Clang toolchain? Well, please install it then and come back when you have it, KTHXBYE 😎